spec/commands/newgem_spec.rb

```diff
@@ -144,6 +144,23 @@ def gem_skeleton_assertions(gem_name)
         expect(bundled_app("test_gem/README.md").read).not_to include("github.com/bundleuser")
       end
     end
+
+    context "git is not installed in the system" do
+      before do
+        `rm "#{ENV["GIT_CONFIG"]}"` if File.exist?(ENV["GIT_CONFIG"])
+        reset!
+        in_app_root
+        bundle "gem #{gem_name}"
+        remove_push_guard(gem_name)
+      end
+
+      it "contribute URL set to [USERNAME]" do
+        expect(bundled_app("test_gem/README.md").read).to include("[USERNAME]")
+        expect(bundled_app("test_gem/README.md").read).not_to include("github.com/bundleuser")
```

Comment:
You probably will want to test the `gemspec` as well to make sure we're setting sensible defaults.